source/prerequisite-tasks.js

```diff
@@ -15,6 +16,20 @@ module.exports = (input, pkg, options) => {
 			skip: () => pkg.private || isExternalRegistry,
 			task: async () => npm.checkConnection()
 		},
+		{
+			title: 'Check npm version',
+			task: async () => {
+				const versions = JSON.parse(await execa.stdout('npm', ['version', '--json']));
+
+				if (version.satisfies(versions.npm, '<6.5.0')) {
+					throw new Error(`npm@${versions.npm} does not support enabling two-factor authentication on new repositories`);
```

Comment:
Not sure about the error messages. Should we help the user and tell which version he should install? Currently, only `npm@6.5.0` works. `npm@6.8.0` will probably work as well, but it's not released yet.

Reply:
> `npm@6.8.0` will probably work as well, but it's not released yet.

It is: https://github.com/npm/cli/releases/tag/v6.8.0

I think we should just drop this error message and require npm@6.8.0 for everyone.

Reply:
Done, good idea. When I pushed this yesterday, v6.8 wasn't released yet so I had to add this because otherwise it was untestable :).